protos/v1alpha1/pmap_event.proto

```diff
@@ -44,4 +44,13 @@ message GitHubSource {
 
   // Required. The git commit.
   string commit = 3;
+
+  // Required. The github workflow that triggered the pmap event.
+  string workflow = 4;
+
+  // Required. The sha for the github workflow.
+  string workflow_sha = 5;
```

Comment:
cc: @capri-xiyue 

Should we add workflow run id and run attempt?

Reply:
Yes. That would be useful.

Reply:
I'm not seeing where this was added.

Reply:
@sethvargo this will be added when fixing #94.

Reply:
ack